<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>867</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>person search</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(53, 59, 76);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="viode_frame">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="video_show_label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>-10</y>
           <width>641</width>
           <height>421</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Video Show Window</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="query_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="query_image_label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>0</y>
           <width>601</width>
           <height>411</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>query image</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="button_frame">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string>color: rgb(255, 255, 255);
background-color: rgb(67, 78, 108);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>34</y>
           <width>591</width>
           <height>329</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>40</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="load_video_button">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(66, 77, 106);
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>load video</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="camer_button">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(66, 77, 106);
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>camera</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="load_query_button">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(66, 77, 106);
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>load query</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="load_yolo_model_button">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(66, 77, 106);
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>load yolo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="load_reid_model_button">
              <property name="text">
               <string>load reid</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="select_model_button">
              <property name="text">
               <string>select model</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="detect_button">
              <property name="text">
               <string>detect</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="load_model_label">
            <property name="text">
             <string>waiting model loading....</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="author_label">
            <property name="text">
             <string>author:尹以鹏</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="wechat_label">
            <property name="text">
             <string>Wechat:y24065939s</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEdit">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="reslut_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="search_resluts_label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>0</y>
           <width>601</width>
           <height>411</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(67, 78, 108);</string>
         </property>
         <property name="text">
          <string>search resluts</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
